TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Shooter.java

```diff
@@ -0,0 +1,69 @@
+package org.firstinspires.ftc.teamcode;
+
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.DcMotorSimple;
+import com.qualcomm.robotcore.hardware.Gamepad;
+
+@TeleOp(name = "Shooter", group = "LinearOpMode")
```

Comment:
Something to do in a later PR, we should bring this code together with the mecanum drive, so we don't need to specify it's own tele-op